### DISCUSSION TITLE
Pydra hack week

### DISCUSSION THREAD

**Discussion by djarecka** [nipype/pydra#624] [Polls]
As we discussed last Monday it would be nice to have a hack week that we can focus on a few key issues.

This will be a virtual event (unless someone wants to come to MIT) with short daily meeting to report the progress and discuss the issues

**The poll doesn't have an option for multiple answers, so you can use the poll to mark the best option for you, and in the comments you can write which other weeks can work for you, or  which other weeks will absolutely not work for you**

**Comment by ghisvail** [comment]
Sadly, I can't do the week of April 24-48. I picked my preference for the first week of April, but would be happy with other dates.

**Comment by yibeichan** [comment]
Apr. 3-7 is the first week of our spring quarter (teaching duty). Apr. 24-28 has a conference. I chose Apr. 10-14 but would be okay with whatever the final date is (I can move things around and make it work)

**Comment by tclose** [comment]
I picked the final week, May 8-12, as it is the only full one without a public holiday in Oz. May 1-5 would be my next preference.

I won't be available much over the school holidays here, April 7 - 26.

The first week, April 3-7, would be okay apart from Good Friday. Although since I'm almost a day ahead I will probably only overlap with you all for the middle 3 days.

**Comment by djarecka** [comment]
Sorry for asking you again, but I think the github poll is not the best tool to create this poll. I've created another poll using doodle and I decided to give the daily options. Perhaps we won't be able to choose a week, but 2 days in one week and 2 days in the following week?
https://doodle.com/meeting/organize/id/bqQOyqDb

**Reply by djarecka** [reply]
@effigies - can I ask you to fill the doodle poll, thanks!

**Reply by effigies** [reply]
Yes, sorry.

**Reply by effigies** [reply]
Doodle's interface has become really terrible, so I only responded on days when everybody else was available. If you want me to go back and fill in other days, I can, but I assume I'm the last person.

**Reply by djarecka** [reply]
ok, thank you! I will take a look later today and suggest something!

**Comment by djarecka** [comment]
What about doing hacking days 2 weeks in a row on Monday and Tuesday in May? 
May 1st, 2nd and May 8th, 9th

**Reply by ghisvail** [reply]
I like the idea :+1:

**Comment by yibeichan** [comment]
sounds good to me!

**Comment by djarecka** [comment]
Since May 1st and May 8th are holidays in France, we decided to move the hack-days to Tue-Wed:
- May 2nd, 3rd
- May 9th, 10th

Also, today we created a list that we would like to work on during the hack-days:
https://docs.google.com/document/d/1X9SWSqlqKnym7-n9VBVFvtlAhqhgCyTqTL9XIrIIM9s/edit#
